src/utils/dashboard.js

```diff
@@ -50,30 +50,26 @@ export async function serverAddress() {
     console.debug('URL candidates:', urls);
 
     const promises = urls.map(url => {
-        return fetch(`${url}/System/Info/Public`).then(resp => {
-            return {
-                url: url,
-                response: resp
-            };
-        }).catch(() => {
-            return Promise.resolve();
-        });
+        return fetch(`${url}/System/Info/Public`)
+            .then(async resp => {
+                if (!resp.ok) {
+                    return;
+                }
+
+                return {
+                    url: url,
+                    config: await resp.json()
```

Comment:
Since we are using async/await any errors thrown will be handled by the `.catch`